src/components/Communication.js

```diff
@@ -13,7 +13,14 @@ class Communication {
     // Creates and logs in a user to the server.
     this.ds = createDeepstream(options.host_ip);
     this.id = this.ds.getUid();
-    this.client = this.ds.login({ username: this.id }, this.onLoggedIn.bind(this));
+    this.client = this.ds.login(
+      {
+        username: this.id,
+        id: 'tddd96client',
```

Comment:
id and password is hardcoded into the code.
Put these inside the config.js file.